bin/migrate_airflow_db.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
```

Comment:
Fail if variables are not defined.

```suggestion
set -u

```